src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -446,6 +492,19 @@ private IConfidentialClientApplication BuildConfidentialClientApplication()
             }
         }
 
+        private async Task<string> GetAccessTokenFromApplicationCacheAsync(
```

Comment:
This method name is misleading - in case of a cache miss this goes to Evo. nit: I don't think it's even needed, it's just one line of code.

Reply:
have removed. i was hoping to isolate stuff to test, but doesn't make sense at this point.

---
In reply to: [385597157](https://github.com/AzureAD/microsoft-identity-web/pull/39#discussion_r385597157) [](ancestors = 385597157)